PathTracking/stanley_controller/stanley_controller.py

```diff
@@ -128,17 +139,21 @@ def calc_target_index(state, cx, cy):
     fy = state.y + L * np.sin(state.yaw)
 
     # Search nearest point index
-    dx = [fx - icx for icx in cx]
-    dy = [fy - icy for icy in cy]
-    d = [np.sqrt(idx ** 2 + idy ** 2) for (idx, idy) in zip(dx, dy)]
-    error_front_axle = min(d)
-    target_idx = d.index(error_front_axle)
-
-    target_yaw = normalize_angle(np.arctan2(fy - cy[target_idx], fx - cx[target_idx]) - state.yaw)
+    squared_error_of_front_axle = np.inf
```

Comment:
@wlbksy I suppose you should not use for loop in Python. I think the list comprehension and min function is faster than it. what do you think?

Reply:
I just tested and You're right. I'll revert this. Sorry for the trouble

Reply:
You don't need to say sorry : ).

Reply:
When you commit it, I will merger your PR.